WordPress/src/test/java/org/wordpress/android/ui/posts/PostSettingsUtilsTest.kt

```diff
@@ -119,9 +124,11 @@ class PostSettingsUtilsTest : BaseUnitTest() {
     @Test
     fun `returns "published on" for local draft when date is not set`() {
         postModel.setIsLocalDraft(true)
-        val calendar = Calendar.getInstance()
-        calendar.add(Calendar.MINUTE, -5)
-        postModel.setDateCreated(DateTimeUtils.iso8601FromDate(calendar.time))
+
```

Comment:
- The name of this tests says "published on" but the assertion checks for "publish on"
- The name of this tests says the date is not set, but it seems we set the date to "2019-05-05T20:28:20+0200", right?

Reply:
Thanks for pointing this out. I will check this out.

Reply:
Exactly. I don't understand this test or the purpose of it so I removed it 9d8e5de but I am now thinking that since the `Publish on` logic exists, a test should verify the conditions that can cause it. I just don't understand it's placement.

Reply:
I definitely wouldn't remove it just because we don't understand it. 
1. I'd check the history if the name/content was changed after the original author created it
2. I'd ask the original author what they think about it

My guess is that the name should be "returns "publish on" for local draft when date within the next 30 minutes".

Reply:
Thanks for this @malinajirka these are solid tips that I will keep in my checklist for knowing how to navigate these scenarios.

> My guess is that the name should be "returns "publish on" for local draft when date within the next 30 minutes".

Nice! This is it. It all makes sense now 🙏

Reply:
Resolved here 2ef44ee